src/test/java/org/fundacionjala/dojo/ValidatorTest.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.dojo;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+
+/**
+ * Test for {@link Validator}
+ */
+public class ValidatorTest {
+    private Validator validator;
+
+    @Before
+    public void setUp() {
+        validator = new Validator();
```

Comment:
Is it necessary to have an instance for Validator?

I don't think so, because your class doesn't have a constructor